frontend/src/features/admin-form/create/workflow/components/WorkflowContent/EditStepBlock/EditStepBlockContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import { Stack } from '@chakra-ui/react'
+
+export const EditStepBlockContainer = ({
+  children,
+}: {
+  children?: React.ReactNode
+}): JSX.Element => {
+  return (
+    <Stack
+      direction="column"
+      spacing="0.75rem"
+      px={{ base: '1.5rem', md: '2rem' }}
+      borderTopColor="secondary.200"
```

Comment:
Is this used? Didn't see any accompanying `borderTopWidth` or `borderWidth`

Reply:
thanks for catching that `borderTopColor` can be removed!  

For the others, it is still being used.
For more elaboration: 
- without `px` 
<img width="571" alt="image" src="https://github.com/user-attachments/assets/8586a4a4-04e7-4b29-a640-9b7223ac1be9">

- for `spacing`, it is used to set the default minimum 16px spacing between the items, unless explicitly overriden by components which want more spacing.